docs/manual/working/scalaGuide/main/tests/code/allbrowserspersuite/ExampleOverrideBrowsersSpec.scala

```diff
@@ -5,37 +5,43 @@ package scalaguide.tests.scalatest.allbrowserspersuite
 
 import org.scalatestplus.play._
 import org.scalatestplus.play.guice.GuiceOneServerPerSuite
-import play.api.mvc._
-import play.api.inject.guice._
-import play.api.routing._
-import play.api.routing.sird._
-import play.api.cache.ehcache.EhCacheModule
+import play.api._
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.mvc.DefaultActionBuilder
 
 // #scalafunctionaltest-allbrowserspersuite
 class ExampleOverrideBrowsersSpec extends PlaySpec with GuiceOneServerPerSuite with AllBrowsersPerSuite {
 
   override lazy val browsers =
     Vector(
       FirefoxInfo(firefoxProfile),
-      ChromeInfo
+      ChromeInfo()
     )
 
   // Override app if you need an Application with other than
   // default parameters.
-  override def fakeApplication() =
-    new GuiceApplicationBuilder().disable[EhCacheModule].router(Router.from {
```

Comment:
It's great that you no longer need to remove the EhCacheModule in every single test!